From 32f6adc7182f3c048b5f650702baad5d18f44ef2 Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Thu, 5 May 2011 16:26:31 +0800
Subject: [PATCH 140/635] ENGR00141162 Fix the rotated direction has 90 degree
 deviation issue.

Original design sensor HAL with ro.hardware.hwrotation 270 degree.
Now we will use default screen to portrait with 90 degree rotation
by defining ro.sf.hwrotation. So needs to change CONVERT_A_X and
CONVERT_A_Y definition.

Please also change this definition if ro.sf.hwrotation is updated.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 libsensors/sensors.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libsensors/sensors.h b/libsensors/sensors.h
index 212bf09..10ca250 100755
--- a/libsensors/sensors.h
+++ b/libsensors/sensors.h
@@ -75,8 +75,8 @@ __BEGIN_DECLS
 // conversion of acceleration data to SI units (m/s^2)
 #define RANGE_A                     (2*GRAVITY_EARTH)
 #define CONVERT_A                   (GRAVITY_EARTH / LSG)
-#define CONVERT_A_X                 (-CONVERT_A)
-#define CONVERT_A_Y                 (CONVERT_A)
+#define CONVERT_A_X                 (CONVERT_A)
+#define CONVERT_A_Y                 (-CONVERT_A)
 #define CONVERT_A_Z                 (-CONVERT_A)
 
 // conversion of magnetic data to uT units
-- 
1.8.0

